### PR TITLE
Update to latest submodules

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**
(Blocked until [dhtproto neo-alpha-4](https://github.com/sociomantic-tsunami/dhtproto/milestone/3) is released.)

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Updated to use the release neo-alpha-4. Unblocked.

**Comment by nemanja-boric-sociomantic** [comment]
Travis says no.

**Comment by gavin-norman-sociomantic** [comment]
Weird...
```
./src/dhtnode/request/neo/GetAll.d(173): Error: function dhtnode.request.neo.GetAll.GetAllImpl_v0.getNext does not override any function, did you mean to override 'dhtproto.node.neo.request.GetAll.GetAllProtocol_v0.getNext'?
```
After `d2conv`:
```D
    // GetAllProtocol_v0:
    abstract protected bool getNext (
        scope void delegate ( hash_t key, Const!(void)[] value ) dg );

    // GetAllImpl_v0 : GetAllProtocol_v0:
    override protected bool getNext (
        scope void delegate ( hash_t key, Const!(void[]) value ) dg )
```
Looks ok to me... :/

**Comment by gavin-norman-sociomantic** [comment]
Updated.

**Comment by gavin-norman-sociomantic** [comment]
Happy D2 day, Travis.

File: src/dhtnode/node/RequestHandlers.d [hunk 1]
```diff
@@ -38,4 +39,5 @@ static this ( )
     requests.add(RequestCode.Get, "Get", &Get.handle);
     requests.add(RequestCode.Mirror, "Mirror", &Mirror.handle);
     requests.add(RequestCode.GetAll, "GetAll", &GetAll.handle);
+    requests.add(RequestCode.GetChannels, "GetChannels", &GetChannels.handle);
```
Comment (nemanja-boric-sociomantic) [inline src/dhtnode/node/RequestHandlers.d]: You don't want this bit (+ all `GetChannel` files in the node) as a separate commit?

Comment (gavin-norman-sociomantic) [reply src/dhtnode/node/RequestHandlers.d]: Well, it's required for the dhtproto tests to pass.

Comment (gavin-norman-sociomantic) [reply src/dhtnode/node/RequestHandlers.d]: So I thought I'd put it all in one commit, along with the submodule update.

Comment (nemanja-boric-sociomantic) [reply src/dhtnode/node/RequestHandlers.d]: OK, that's a fair point, I've missed that.

**Pull request merged**